Makefile

```diff
@@ -89,15 +74,6 @@ DOCKER_CONFIG ?= $(HOME)/.docker/config.json
 escapefs = $(subst :,---,$(subst /,___,$(1)))
 unescapefs = $(subst ---,:,$(subst ___,/,$(1)))
 
-# Targets used when cross building.
-.PHONY: register
```

Comment:
The updated `Dockerfile` doesn't need any coreutils to run so qemu isn't needed.